src/runtime/components/forms/Range.vue

```diff
@@ -92,7 +92,8 @@ export default defineComponent({
   setup (props, { emit }) {
     const { ui, attrs, attrsClass } = useUI('range', props.ui, config)
 
-    const { emitFormChange, inputId, color, size, name } = useFormGroup(props, config)
+    const { emitFormChange, inputId, color, size: formGroupSize, name } = useFormGroup(props, config)
+    const size = computed(() => ui.value.size[formGroupSize] ? formGroupSize : (props.size ?? config.default?.size))
```

Comment:
I recently refactored how FormGroup props are injected (https://github.com/nuxt/ui/pull/704), it might be better to do this check for all inputs by doing here: https://github.com/nuxt/ui/blob/dev/src/runtime/composables/useFormGroup.ts#L47

Also, `props.size` should have priority over `formGroupSize`

I guess something like this should work?
```ts
# useFormGroup.ts
export const useFormGroup = (...) => {
    return {
      // ...
      size: computed(() => {
        const formGroupSize = config.sizes[formGroup?.size.value] ? formGroup?.size.value : undefined
        return inputProps?.size ?? formGroupSize ?? config?.default?.size
      }),
      // ...
    }
}
```

Reply:
@romhml , could you 2-check the last commit, pls?
And I think we really should add types to the `ui.config.ts` - file..... It would be great to use types inside `useUI` or `useFormGroup` instead of any. Am I wrong, or i can create an issue for it?